node/src/BaseClient.ts

```diff
@@ -3416,6 +3417,33 @@ export class BaseClient {
         );
     }
 
+    /**
+     * Returns the positions (longitude, latitude) of all the specified `members` of the
+     * geospatial index represented by the sorted set at `key`.
+     *
+     * See https://valkey.io/commands/geopos for more details.
+     *
+     * @param key - The key of the sorted set.
+     * @param members - The members for which to get the positions.
+     * @returns A 2D `array` which represents positions (longitude and latitude) corresponding
```

Comment:
```suggestion
     * @returns A 2D `Array` which represents positions (longitude and latitude) corresponding
```

Should the "array" start with Capital A?